webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -7355,6 +7359,11 @@ var CURRENT_GROUP_DATA={};
                 };
                 return conversationDetail;
             };
+            _this.checkWarningBox = function () {
+                if(!(warningBox.classList.contains('n-vis'))){
```

Comment:
@hardikparashar  Avoid using if statements if possible 
Check if this works **!(warningBox.classList.contains('n-vis') && warningBox.classList.add('n-vis');**